paymentsheet/src/main/java/com/stripe/android/paymentsheet/repositories/CustomerApiRepository.kt

```diff
@@ -129,4 +130,18 @@ internal class CustomerApiRepository @Inject constructor(
         ).onFailure {
             logger.error("Failed to attach payment method $paymentMethodId.", it)
         }
+
+    override suspend fun updatePaymentMethod(
+        customerConfig: PaymentSheet.CustomerConfiguration,
```

Comment:
Let’s pass in only the customer EK.

Reply:
This does break the pattern of `CustomerConfiguration` being passed across each modification method. Are we okay with that?